releasenotes/notes/cphase-rzz-equivalence-e8afc37b71a74366.yaml

```
@@ -0,0 +1,6 @@
+---
+features_circuits:
+  - |
+    New equivalence rules in the standard equivalence library mean that the transpiler can now
+    directly convert between two-qubit continuous Pauli rotations, rather than always decomposing
```

Did it always decompose into CX? There are some existing 2q Pauli equivalences that should allow to convert between some two-qubit rotations, no? 🙂 
```suggestion
    directly convert between two-qubit continuous Pauli rotations, rather than sometimes decomposing
```

---

I'll tweak this note - I wrote it under the same false assumption about how the basis translator works that's alluded to in the PR comment. We need a new translation algorithm to make _any_ choice of translation reliable - the current system is good at finding an arbitrary valid translation, but not at finding _the best_ translation (for any definition of "best").

---

I've written a new version of the release note in bbe529b604.